src/hotspot/share/interpreter/oopMapCache.hpp

```diff
@@ -88,11 +89,9 @@ class InterpreterOopMap: ResourceObj {
   unsigned short _bci;            // the bci    for which the mask is valid
 
  protected:
-#ifdef ASSERT
-  bool _resource_allocate_bit_mask;
-#endif
-  int            _num_oops;
-  intptr_t       _bit_mask[N];    // the bit mask if
+  DEBUG_ONLY(bool _used;)
```

Comment:
Minor nit. This changes memory layout between debug and release builds, and this is used as part of OopMapCache. Not a big concern, but I usually prefer having the same layout between debug and release to test what we ship.

Can't we not just assert that mask size == USHRT_MAX?

Reply:
Yes, fixed.